test/transcoding/OpImageSampleExplicitLod_arg.ll

```diff
@@ -51,7 +51,7 @@ entry:
 
   store <4 x float> %call1, <4 x float> addrspace(1)* %results, align 16
   %call2 = call spir_func <4 x float> @_Z11read_imagef14ocl_image2d_ro11ocl_samplerDv2_fDv2_fDv2_f(%opencl.image2d_ro_t addrspace(1)* %image, %opencl.sampler_t* %imageSampler, <2 x float> %coord, <2 x float> %dx, <2 x float> %dy)
-; CHECK-LLVM: call spir_func <4 x float> @_Z11read_imagef14ocl_image2d_ro11ocl_samplerDv2_fS_S_(%opencl.image2d_ro_t addrspace(1)* %image, %opencl.sampler_t* %imageSampler, <2 x float> %coord, <2 x float> %dx, <2 x float> %dy)
+; CHECK-LLVM: call spir_func <4 x float> @_Z11read_imagef14ocl_image2d_ro11ocl_samplerDv2_fS{{[0-9]*}}_S{{[0-9]*}}_(%opencl.image2d_ro_t addrspace(1)* %image, %opencl.sampler_t* %imageSampler, <2 x float> %coord, <2 x float> %dx, <2 x float> %dy)
```

Comment:
This was actually a bug in the translator that has been fixed by `ed99fc0 ("Fix mangling substitution for primitive types", 2019-06-03)`.  If you want it on the 7.0 branch then we should probably cherry-pick that commit instead?

Reply:
You are right. Commit ed99fc0 cherrypicked to 70 branch and my changes rebased.